tools/reviewer/job/sync/tests/LineNumberConverterTest.java

```diff
@@ -214,74 +201,99 @@ public void changedThreeLines_lastChangedLineOnTheRightTest() {
+    assertEquals(
+        41,
+        LineNumberConverter.getPosition(COMPLEX_CHANGES_PATCH, 38, LineNumberConverter.Side.LEFT));
   }
 
   @Test
   public void complexChanges_line65_OnTheLeftTest() {
-    LineNumberConverter converter = new LineNumberConverter(COMPLEX_CHANGES_PATCH);
-
-    assertEquals(65, converter.getLineNumber(54, LineNumberConverter.Side.LEFT));
-    assertEquals(54, converter.getPosition(65, LineNumberConverter.Side.LEFT));
+    assertEquals(
+        65,
+        LineNumberConverter.getLineNumber(
+            COMPLEX_CHANGES_PATCH, 54, LineNumberConverter.Side.LEFT));
+    assertEquals(
+        54,
+        LineNumberConverter.getPosition(COMPLEX_CHANGES_PATCH, 65, LineNumberConverter.Side.LEFT));
   }
 
   @Test
   public void complexChanges_line100_OnTheLeftTest() {
-    LineNumberConverter converter = new LineNumberConverter(COMPLEX_CHANGES_PATCH);
-
-    assertEquals(100, converter.getLineNumber(81, LineNumberConverter.Side.LEFT));
-    assertEquals(81, converter.getPosition(100, LineNumberConverter.Side.LEFT));
+    assertEquals(
+        100,
+        LineNumberConverter.getLineNumber(
+            COMPLEX_CHANGES_PATCH, 81, LineNumberConverter.Side.LEFT));
+    assertEquals(
+        81,
+        LineNumberConverter.getPosition(COMPLEX_CHANGES_PATCH, 100, LineNumberConverter.Side.LEFT));
   }
 
   @Test
   public void complexChanges_line18_OnTheRightTest() {
-    LineNumberConverter converter = new LineNumberConverter(COMPLEX_CHANGES_PATCH);
-
-    assertEquals(18, converter.getLineNumber(24, LineNumberConverter.Side.RIGHT));
-    assertEquals(24, converter.getPosition(18, LineNumberConverter.Side.RIGHT));
+    assertEquals(
+        18,
+        LineNumberConverter.getLineNumber(
+            COMPLEX_CHANGES_PATCH, 24, LineNumberConverter.Side.RIGHT));
+    assertEquals(
+        24,
+        LineNumberConverter.getPosition(COMPLEX_CHANGES_PATCH, 18, LineNumberConverter.Side.RIGHT));
   }
 
   @Test
   public void complexChanges_line30_OnTheRightTest() {
-    LineNumberConverter converter = new LineNumberConverter(COMPLEX_CHANGES_PATCH);
-
-    assertEquals(30, converter.getLineNumber(35, LineNumberConverter.Side.RIGHT));
-    assertEquals(35, converter.getPosition(30, LineNumberConverter.Side.RIGHT));
+    assertEquals(
+        30,
+        LineNumberConverter.getLineNumber(
+            COMPLEX_CHANGES_PATCH, 35, LineNumberConverter.Side.RIGHT));
+    assertEquals(
+        35,
+        LineNumberConverter.getPosition(COMPLEX_CHANGES_PATCH, 30, LineNumberConverter.Side.RIGHT));
   }
 
   @Test
   public void complexChanges_line100_OnTheRightTest() {
-    LineNumberConverter converter = new LineNumberConverter(COMPLEX_CHANGES_PATCH);
-
-    assertEquals(100, converter.getLineNumber(80, LineNumberConverter.Side.RIGHT));
-    assertEquals(80, converter.getPosition(100, LineNumberConverter.Side.RIGHT));
+    assertEquals(
+        100,
+        LineNumberConverter.getLineNumber(
```

Comment:
Just so you know, even if there's a `static` method, it's _fine_  to still create an object if using many times.
Your choice, just saying that both ways can exist together in the codebase.

Reply:
Thank you!
Done.